tests/kerastuner/engine/hyperparameters_test.py

```diff
@@ -94,10 +95,10 @@ def test_conditional_scope():
         child1 = hp.Choice('child_choice', [4, 5, 6])
```

Comment:
What if the parent with name 'choice' has no default? In that case there will be no active parameters. Would value then return an empty dict (possibly test that too)?

Reply:
In Keras Tuner, hyperparameters always have a default. If no explicit `default` is passed for `hp.Choice`, then the first item is used (i.e. `4` here). This is necessary to support inlining of search space definitions